.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
please remove this file.

Reply:
Okay. I will. I have added it to my gitignore already too

Reply:
Its already 19 hours!! @igbominadeveloper

Reply:
I fixed it since yesterday. @profmcdan